dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java

```diff
@@ -50,6 +51,19 @@ public static File getBundleRoot(String name, boolean createDir) {
 		return dir;
 	}
 
+    public static File getStaticBundleRoot(String name) {
+        return getStaticBundleRoot(name, true);
+    }
+
+    public static File getStaticBundleRoot(String name, boolean createDir) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "createDir" to determine which action to take. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2301)